<?xml version="1.0" encoding="utf-8"?>

<launch>

  <!-- Load Parameter -->
  
  <param name="yolov5_path"       value="$(find yolov5_ros)/yolov5"/>
  <param name="use_cpu"           value="true" />


  <node pkg="yolov5_ros" type="bluedecectdemo.py" name="yolov5_ros" output="screen" >
    <param name="weight_path"       value="$(find yolov5_ros)/weights/bestblue1248silos.pt"/>
    <param name="image_topic"       value="/camera/image_raw" />
    <param name="pub_topic1"         value="/yolov5/silos" />
    <param name="pub_topic2"         value="/yolov5/judge" />
    <param name="camera_frame"       value="camera_color_frame"/>
    <param name="conf"              value="0.3" />
   
  </node>

  <!-- <node name="camera_ros" pkg="yolov5_ros" type="camera.py" output="screen"> -->
  <node pkg="yolov5_ros" type="camera.py" name="camera_ros" output="screen" >
   <param name="judge_topic1"       value="/cam_cov"/>
  </node>


</launch> 

